documentation/user_docs/docs/conf.py

```diff
@@ -34,6 +37,14 @@
     "sphinx.ext.autosectionlabel",
 ]
 
+source_suffix = {
+    '.rst': 'restructuredtext',
+    '.txt': 'restructuredtext',
```

Comment:
Are `.txt` files reStructuredText?

Reply:
Mostly based on the sphinx guidelines that you should use ".txt" rather than ".rst" for obscure reasons. I can remove it.

Reply:
If it's following sphinx guidelines I'd leave it in 👍